prime-router/src/main/kotlin/fhirengine/utils/FHIRBundleHelpers.kt

```diff
@@ -169,4 +179,52 @@ object FHIRBundleHelpers {
             is Observation -> getDiagnosticReportNoObservations(this).forEach { this.deleteResource(it) }
         }
     }
+
+    /**
+     * Gets the observation extensions for those observations that pass the condition filter, unless they all pass for
+     * a diagnostic report
+     * @param fhirBundle the bundle will hold the actual value that %testPerformedCodes will evaluate to
```

Comment:
Try to not use @ param. This was a decision by the program a while back. Use the [] in the description